[IN-152] Automox XSOAR Documentation (#3)
ax-jmiller/content: Packs/Automox/Integrations/Automox/README.md

> Use the Automox integration to create device groups, update devices, run policies, and remediate vulnerabilities of devices in your Automox instance.

Reviewer: ```suggestion
Use the Automox integration to create device groups, update devices, run policies, and remediate vulnerabilities of devices through the Automox platform.
```

Author: OOh, I'm saving this verbiage in my pocket. Committed.

> This integration was integrated and tested with version 2022-02-24 of Automox

Reviewer: `as of` instead of `with version`?

Author: Let's try this.

```suggestion
This integration was integrated and tested as of 2022-03-21 with the Automox API
```
